pkg/klusterlet/clusterinfo/clusterinfo_controller.go

```diff
@@ -597,6 +667,12 @@ func (r *ClusterInfoReconciler) GetClusterClaims() ([]*clusterv1alpha1.ClusterCl
 	if len(distributionInfo.OCP.Version) > 0 {
 		claims = append(claims, newClusterClaim("version.openshift.io", distributionInfo.OCP.Version))
 	}
+
+	infraName := r.getInfraName()
+	if len(infraName) > 0 {
+		claims = append(claims, newClusterClaim("infrastructure.openshift.io", infraName))
```

Comment:
for infrastructure, we may use a json string, e.g. `{"infraName":"cluster-1234"}`

Reply:
done
```
- name: infrastructure.openshift.io
    value: '{"infraName":"yzw-8tlfw"}'
- name: region.open-cluster-management.io
    value: us-east-1
```